HISTORY.rst

```diff
@@ -53,6 +53,8 @@ Unreleased Changes
     * Options' display names will now be shown in dropdown menus
       (`#1217 <https://github.com/natcap/invest/issues/1217>`_)
     * Represent boolean inputs with a toggle switch rather than radio buttons.
+    * Includes local versions of the User Guide in English, Spanish, & Chinese.
```

Comment:
I think it was discussed, but I can't remember, is it okay to call it a "Chinese" translation?

Reply:
I was told Chinese is fine, we do not need to specify Mandarin.